exporter/coralogixexporter/example/config-example-1.yaml

```diff
@@ -1,20 +1,27 @@
 receivers:
-    otlp:
-        protocols:
-            grpc:
-            http:
+    opencensus:
```

Comment:
Why opencensus?

Reply:
This is an example of the configuration file for metrics.
Opencensus is just an example of a metrics reciver.

Reply:
Can't you use OTLP here? OpenCensus will eventually be deprecated.

Reply:
Ok, I will change it to "OTLP" receiver